beacon-chain/core/helpers/committee.go

```diff
@@ -511,11 +329,11 @@ func CommitteeAssignment(
 // Spec pseudocode definition:
 // 	def get_shard_delta(state: BeaconState, epoch: Epoch) -> int:
 //    return min(get_epoch_committee_count(state, epoch), SHARD_COUNT - SHARD_COUNT // SLOTS_PER_EPOCH)
-func ShardDelta(beaconState *pb.BeaconState, epoch uint64) uint64 {
```

Comment:
maybe have another name other than state ? state is the name of one of our core packages too.

Reply:
renamed it back to `beaconState`